taxonomies/food/ingredients.txt

```diff
@@ -37921,37 +37826,6 @@ wikipedia:en: https://en.wikipedia.org/wiki/Potassium_phosphate
 # mineral/potassium-phosphate has 123 products @2018-1106
 # additive/e340-potassium-phosphates has 2576 products in 18 languages @2018-11-06
 
-# description:en:Monopotassium phosphate, MKP, (also potassium dihydrogenphosphate, KDP, or monobasic potassium phosphate), KH2PO4, is a soluble salt of potassium and the dihydrogen phosphate ion
-
-< en:potassium phosphates
```

Comment:
This is only one block, but actually there are a couple of block after this one that seems to be present in both additives and here in food.

Can someone confirm that we want to have them in a single place? And that this place should be additives?

Reply:
yes, it should be additives. there could be a few tricky ones, e.g. soy lecithin, where we want it under soy